src/EnergyPlus/WaterUse.cc

```diff
@@ -1092,117 +1099,142 @@ namespace WaterUse {
-                    ++this->CWHWTempErrorCount;
-                    if (this->CWHWTempErrorCount < 2) {
+                    // print error for variables of target water temperature
+                    ++this->TargetCWTempErrorCount;
+                    TempDiff = this->ColdTemp - this->TargetTemp;
+                    if (this->TargetCWTempErrorCount < 2) {
                         ShowWarningError(
                             state,
-                            format("CalcEquipmentFlowRates: \"{}\" - Hot water temperature is less than the cold water temperature ({:.2R} C)",
+                            format("CalcEquipmentFlowRates: \"{}\" - Target water temperature is less than the cold water temperature by ({:.2R} C)",
                                    this->Name,
                                    TempDiff));
                         ShowContinueErrorTimeStamp(state, "");
-                        ShowContinueError(state, format("...hot water temperature       = {:.2R} C", this->HotTemp));
+                        ShowContinueError(state, format("...target water temperature     = {:.2R} C", this->TargetTemp));
                         ShowContinueError(state, format("...cold water temperature       = {:.2R} C", this->ColdTemp));
-                        ShowContinueError(state, "...Note: hot water temperature should be greater than or equal to the cold water temperature");
                         ShowContinueError(state,
-                                          "...Hot water temperature should be greater than or equal to the cold water temperature. "
+                                          "...Target water temperature should be greater than or equal to the cold water temperature. "
                                           "Verify temperature setpoints and schedules.");
                     } else {
                         ShowRecurringWarningErrorAtEnd(
                             state,
-                            format("\"{}\" - Hot water temperature should be greater than or equal to the cold water temperature error continues...",
-                                   this->Name),
-                            this->CWHWTempErrIndex,
+                            format(
+                                "\"{}\" - Target water temperature should be greater than or equal to the cold water temperature error continues...",
+                                this->Name),
+                            this->TargetCWTempErrIndex,
                             TempDiff,
                             TempDiff);
                     }
                 }
-            } else if (this->TargetTemp > (this->HotTemp + EPSILON)) {
-                // Target temp is hotter than the hot water temp; can't meet target temperature
+            } else if (this->TargetTemp > this->HotTemp - EPSILON) {
+                // don't need to mix (use hot) or cold water flow would be very small (use hot), or need to purge stagnant hot water temps (use hot)
```

Comment:
Gotta think if target temp > hot temp then use hot water flow. Maybe there are corner cases, idk.

Reply:
Changed from (this->HotTemp - EPSILON) to (this->HotTemp + EPSILON) to minimize diffs (same 2 diffs as #9651). This should be - EPSILON (or don't use EPSILON) to ensure the target temp is between cold and hot temp. If target temp is slightly > hot temp (i.e., greater by less than 0.001), then the fraction (this->TargetTemp - this->ColdTemp) / (this->HotTemp - this->ColdTemp) will be > 1 (e.g., 1.00009). I tried getting rid of EPSILON and got lot's of diffs but then that is expected. The issue is to be sure that the multiplier below is between 0 and 1.

    this->HotMassFlowRate = this->TotalMassFlowRate * (this->TargetTemp - this->ColdTemp) / (this->HotTemp - this->ColdTemp);

Reply:
Nope, I am getting an assert with this code the way it is. Have to change it and take the hit on diffs.

![image](https://user-images.githubusercontent.com/5778156/190932826-746e41fe-129d-4c1e-a40c-3a51d02a9f78.png)